recipe/build.sh

```diff
@@ -5,6 +5,7 @@ autoreconf -fi
 # Disable link time optimizations as it results in symbols being dropped in libffpack.so
 export LDFLAGS=$(echo "${LDFLAGS}" | sed "s/-Wl,-O2//g")
 export LDFLAGS=$(echo "${LDFLAGS}" | sed "s/-Wl,--as-needed//g")
+export CXXFLAGS=$(echo "${CXXFLAGS}" | sed "s/-std=c++17/-std=c++11/g")
```

Comment:
you might need -std=gnu++11

I'm just basing this off the line in the patch.

Reply:
https://github.com/conda-forge/fflas-ffpack-feedstock/blob/master/recipe/patches/dont-sort-cxxflags.patch#L9